plugin_tests/cached_tiles_test.py

```diff
@@ -47,33 +48,32 @@ def tearDownModule():
 
 class LargeImageCachedTilesTest(common.LargeImageCommonTest):
     def _monitorTileCounts(self):
-        if hasattr(self, 'tileCounter'):
-            return
-        from girder.plugins.large_image.tilesource.test import TestTileSource
-        originalGetTile = TestTileSource.getTile
-        originalWrapKey = TestTileSource.wrapKey
-        self.tileCounter = 0
-        self.keyPrefix = str(time.time())
+        from girder.plugins.large_image.tilesource import test
+        test.tileCounter = 0
 
-        def countGetTile(ttsself, *args, **kwargs):
-            # Increment the counter after the call, so that exceptions won't
-            # increment it.
-            result = originalGetTile(ttsself, *args, **kwargs)
-            self.tileCounter += 1
-            return result
+        self.originalWrapKey = test.TestTileSource.wrapKey
+        self.keyPrefix = str(time.time())
 
         def wrapKey(*args, **kwargs):
             # Ensure that this test has unique keys
-            return self.keyPrefix + originalWrapKey(*args, **kwargs)
+            return self.keyPrefix + self.originalWrapKey(*args, **kwargs)
+
+        test.TestTileSource.wrapKey = wrapKey
```

Comment:
I'm not sure how to use `mock` in this case.  I assume I'd pass my modified wrapKey function as a `new_callable` in some manner, but I'm not sure how to get the class to be correct and have the context last for each test, since there is a separate setUp and tearDown.